operator/internal/securedcluster/values/translation/translation.go

```diff
@@ -158,6 +158,10 @@ func (t Translator) translate(ctx context.Context, sc platform.SecuredCluster) (
 		v.AddChild("network", translation.GetGlobalNetwork(sc.Spec.Network))
 	}
 
+	if sc.Spec.ProcessBaselines != nil {
```

Comment:
Since `AddChild()` is nil-safe then if you make `getAutoLockProcessBaselinesValues` nil-safe, you will be able to make the following line unconditional...

Reply:
Done